Kernel/Heap/Heap.h

```diff
@@ -153,7 +153,7 @@ class Heap {
     size_t m_total_chunks { 0 };
     size_t m_allocated_chunks { 0 };
     u8* m_chunks { nullptr };
-    BitmapView m_bitmap;
+    Bitmap m_bitmap;
```

Comment:
This change doesn't look entirely safe. `Kernel::Heap::m_bitmap` is currently a wrapper around a specific chunk of the allocated heap. This change makes it part of the `Heap` metadata object instead.

I think that's probably fine, it's not like we *have* to stash the bitmap with the heap memory. In fact, separating the metadata from the heap is definitely a good thing!

However, I think this only works right now because `Bitmap` has a `Bitmap(u8*, size_t)` constructor which takes ownership of the passed-in memory. If/when that `Heap` is later deleted, it will call `~Bitmap()` and try to `kfree()` this `u8*` which was not `kmalloc`'ed in the first place.

I think a good fix would be:
1. Remove `Bitmap(u8*, size_t)`
2. Keep this change from `BitmapView` to `Bitmap` but make sure to pass `Heap::m_total_chunks` and `false` to the `Bitmap(size_t, bool)` constructor instead :)

Reply:
Good catch! Fixed.